DropdownAlert.js

```diff
@@ -296,12 +298,15 @@ export default class DropdownAlert extends Component {
   };
   close = (action, onDone = () => {}) => {
     this.animate(0, 250, () => {
-      const { onClose, updateStatusBar, onCancel } = this.props;
+      const { onClose, updateStatusBar, onCancel, onTap } = this.props;
       this.updateStatusBar(updateStatusBar, false);
       this.alertData.action = action;
-      if (action == 'cancel') {
+      if (action == ACTION.cancel) {
         onCancel(this.alertData);
       } else {
+        if (action == ACTION.tap) {
```

Comment:
shouldn't this be?
```
} else if (action == ACTION.tap) {
  onTap(this.alertData);
} else {
  onClose(this.alertData);
}

Reply:
Nah I'm wanting to keep `onClose` for tap action so not to deter those (myself included) who rather keep using the catch all action callback (minus cancel which is deliberate).

Reply:
Ah yes, makes sense. You'd anyway expect that the dropdown closes when tapping and triggering any action.

Reply:
Just finished integration, works really nice.

There's one thing people need to pay attention to if they use the dynamic image source.
For instance, if the default behavior is dropdowns without an image and just a few with dynamic images, _infoImageSrc, errorImageSrc_ and the others should be set to null so that default images aren't shown, only the dynamic ones.

Previously we've had _renderImage_ null so removing that so that we get dynamic images resulted in default images being shown for all messages.

@testshallpass Should we add a new section in the readme file for such info?

Reply:
I see. Can you share snippet of example so I may examine? But, you are right as in need to make it clear that it will override any defined image source props.